app/models/backend/responses/PensionSchemeReturnResponse.scala

```diff
@@ -1,61 +0,0 @@
-/*
- * Copyright 2023 HM Revenue & Customs
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package models.backend.responses
-
-import utils.WithName
-import models.backend.responses.DataEntryRule.{Fixed, Updated}
-import play.api.libs.json._
-
-case class PensionSchemeReturn(
```

Comment:
It's weird that this was added in very early stages, when we had no information about the BE yet. FYI @narendrapaduchuri